src/test/java/specs/FixturesSpec.java

```diff
@@ -483,11 +484,10 @@
               .map((failure) -> failure.getMessage())
               .collect(Collectors.toList()));
 
-          it("report each error", () -> {
-            assertThat(result.get().getFailureCount(), is(2));
+          it("report the error only once", () -> {
```

Comment:
Note: Let's not change this until we come to a decision on the other "to discuss" items.